src/module_object.cpp

```diff
@@ -662,6 +662,27 @@ ArrayObject *ModuleObject::ancestors(Env *env) {
     return ancestors;
 }
 
+bool ModuleObject::ancestors_includes(Env *env, ModuleObject *module) {
+    std::lock_guard<std::recursive_mutex> lock(g_gc_recursive_mutex);
```

Comment:
As described in a bit more detail in the comments: this line can be removed. Once it is gone, I would say it's good to merge.

Reply:
I'm going to investigate this more in #1956. My gut tells me we *do* want the lock here (and in the method below). But I won't hold up this work!

Reply:
Oh, why's that?